lib/Dialect/SPIRV/Serialization/Serializer.cpp

```diff
@@ -131,6 +131,15 @@ class Serializer {
     return funcIDMap.lookup(fnName);
   }
 
+  uint32_t getOrCreateFunctionID(StringRef fnName) {
```

Comment:
We only use inline implementation when it's just a trivial oneliner. Can you move the implementation here out side of the decl?

Reply:
Fixed, thanks!